misc/co2_plume_spawner.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Would you convert these to 4 space indentation?

Reply:
Sorry missed this now done in [fee5a09](https://github.com/BCLab-UNM/dragonfly-sim/pull/8/commits/fee5a096ea67f8bb6e6dc59067d05bc903ab64cf)